.github/workflows/main.yml

```diff
@@ -20,6 +20,11 @@ jobs:
       with:
         ruby-version: head
         bundler-cache: true
+    - name: Install MinGW (Windows)
```

Comment:
I see this is a draft but I wanted to note that I think setup-ruby comes with mingw installed on Windows. This step might not be necessary?

Reply:
Yes, it shouldn't be